app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -0,0 +1,148 @@
+// govuk_frontend_toolkit
+@import 'grid_layout';
+@import 'design-patterns/buttons';
+
+.pub-c-feedback {
+  max-width: 960px;
+  margin: 0 auto;
+}
+
+// hide without js
+// show with js, unless also has the js-hidden class
+.pub-c-feedback__js-show {
+  display: none;
+
+  .js-enabled & {
+    display: block;
+
+    &.js-hidden {
+      display: none;
+    }
+  }
+}
+
+.pub-c-feedback__grid-row {
+  @extend %grid-row;
+}
+
+.pub-c-feedback__column-two-thirds {
+  @include grid-column( 2 / 3 );
+}
+
+.pub-c-feedback__prompt {
+  @extend %contain-floats;
+  background-color: $govuk-blue;
+  color: $white;
+  padding: 10px $gutter-half 0;
+  outline: 0;
+}
+
+.pub-c-feedback__prompt-question,
+.pub-c-feedback__prompt-success {
+  @include bold-16;
+  display: inline;
+
+  &:focus {
+    outline: solid $yellow 3px;
```

Comment:
Could we use a semantic variable here? https://github.com/alphagov/govuk_frontend_toolkit/blob/master/stylesheets/colours/_palette.scss#L63